src/_base/helm/app/templates/application/webapp/deployment.yaml

```diff
@@ -31,7 +31,39 @@ spec:
         {{- end }}
       {{- end }}
       containers:
-      - env:
+      - name: {{ .Values.resourcePrefix }}nginx
+        env:
+        {{- range $key, $value := .Values.docker.services.nginx.environment }}
+        {{- $tp := typeOf $value }}
+        - name: {{ $key | quote }}
+        {{- if eq $tp "string" }}
+          value: {{ tpl $value $ | quote }}
+        {{- else}}
+          value: {{ $value | quote }}
+        {{- end }}
+        {{- end }}
+        image: {{ .Values.docker.image.nginx }}
+        imagePullPolicy: Always
+        ports:
+        - containerPort: 80
+        resources:
+          limits:
+            memory: {{ .Values.resources.limits.memory.nginx }}
+          requests:
+            memory: {{ .Values.resources.requests.memory.nginx }}
+        readinessProbe:
+          tcpSocket:
+            port: 80
+          initialDelaySeconds: 5
+          periodSeconds: 10
+        volumeMounts:
+        {{- if not (eq "" (include "application.volumeMounts.all" .)) }}
+        {{- include "application.volumeMounts.all" . | indent 8 }}
+        {{- else }} []
+        {{- end }}
+
+      - name: {{ .Values.resourcePrefix }}php-fpm
```

Comment:
no need for resource prefix

Reply:
ah ok, this originally had the prefix on (just moved up to the top of the object to make it more visible in a multi-container pod definition), so I added it in the other containers of the pod too